_observing-your-data/prometheusmetrics.md

```diff
@@ -0,0 +1,125 @@
+
+The following image shows the process of pulling in metrics and viewing them in **Observability**.
+
+![Prometheus and Metrics]({{site.url}}{{site.baseurl}}/images/metrics/metricsgif.gif)
+
+## Creating a connection to Prometheus
+
+You can view metrics collected from Prometheus in OpenSearch Dashboards by first creating a connection from [Prometheus](https://prometheus.io/) to OpenSearch using the SQL plugin. 
+
+### Configuring a Prometheus data source connection to OpenSearch
+
+To configure a connection to Prometheus for viewing metrics in OpenSearch Dashboards, create a file on your OpenSearch nodes named `datasources.json` containing the Prometheus data source settings. The following examples demonstrate the various Prometheus data source configurations using different authentication methods:
+
+No authentication:
+
+```json
+[{
+    "name" : "my_prometheus",
+    "connector": "prometheus",
+    "properties" : {
+        "prometheus.uri" : "http://localhost:9090"
+    }
+}]
+```
+
+Basic authentication:
+
+```json
+[{
+    "name" : "my_prometheus",
+    "connector": "prometheus",
+    "properties" : {
+        "prometheus.uri" : "http://localhost:9090",
+        "prometheus.auth.type" : "basicauth",
+        "prometheus.auth.username" : "admin",
+        "prometheus.auth.password" : "admin"
+    }
+}]
+```
+
+AWS SigV4 authentication:
+
+```json
+[{
+    "name" : "my_prometheus",
+    "connector": "prometheus",
+    "properties" : {
+        "prometheus.uri" : "http://localhost:8080",
+        "prometheus.auth.type" : "awssigv4",
+        "prometheus.auth.region" : "us-east-1",
+        "prometheus.auth.access_key" : "{{accessKey}}"
+        "prometheus.auth.secret_key" : "{{secretKey}}"
+    }
+}]
+```
+
+After configuring Prometheus in the `datasources.json` file, run the following command to load the configuration into the OpenSearch keystore. The configuration is securely stored in the keystore, as it contains sensitive credential information.
+
+```
+bin/opensearch-keystore add-file plugins.query.federation.datasources.config datasources.json
+```
+
+If you are updating the keystore during runtime, refresh the keystore using following API command:
+
+```bash
+POST /_nodes/reload_secure_settings
+{
+  "secure_settings_password":""
+  
+}
+```
+{% include copy-curl.html %}
+
+Metrics collected from Prometheus are displayed in the **Observability** application under **Metrics analytics**. For further information, see the [Prometheus Connector](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/admin/prometheus_connector.rst) GitHub page.
```

Comment:
Is this appropriate here? I mean, as soon as we make the connection, are metrics displayed in the observability > metrics analysis window? If so, this would be a good time for a screen shot. :)

Reply:
yes this is accurate. I have a ss of this, so I'll add it back